### ISSUE TITLE
DateTimePickerRow displayed with inline style on iOS 14

### ISSUE THREAD

**Issue by funkenstrahlen** [xmartlabs/Eureka#2089]
The `DateTimePickerRow` row shows the DatePicker as inline picker on iOS 14. This is unexpected and should be exclusive to `DateTimeInlineRow`.

This was changed in #2077.

Eureka 5.3.1
Xcode 12.0.1
iOS 14.0.1

**Comment by mats-claassen** [comment]
You can still use the wheels display by setting:

```swift
.cellSetup({ (cell, row) in
                if #available(iOS 13.4, *) {
                    cell.datePicker.preferredDatePickerStyle = .wheels
                }
            })
```

Do you have any specific reason why the default should be `.wheels` for iOS 14?

**Comment by funkenstrahlen** [comment]
> Do you have any specific reason why the default should be `.wheels` for iOS 14?

In my case because of #2088.

However I also have the feeling it is unexpected behavior in contrast to `DateTimeInlineRow`? But this might just be me and I am open for other opinions.
